AppCenter/AppCenterTests/MSAppCenterIngestionTests.m

```diff
@@ -66,7 +42,7 @@ - (void)setUp {
 
   NSDictionary *headers = @{@"Content-Type" : @"application/json", @"App-Secret" : kMSTestAppSecret, @"Install-ID" : MS_UUID_STRING};
   NSDictionary *queryStrings = @{@"api-version" : @"1.0.0"};
-
+  self.httpClientMock = OCMPartialMock([MSHttpClient new]);
```

Comment:
`stopMocking` for this mock instance in `tearDown`?

Reply:
@jaelim-ms I'm not sure why `stopMocking` method isn't available for `httpClientMock`